README.md

```diff
@@ -1,16 +1,148 @@
 # OpenCensus Specs
 
-## General Design
-* For details about the library structured see [Namespace and Package](https://github.com/census-instrumentation/opencensus-specs/blob/master/NamespaceAndPackage.md)
+This is a high level design of the OpenCensus library, some of the API examples may be written 
+(linked) in C++, Java or Go but every language should translate/modify them based on language 
+specific patterns/idioms. Our goal is that all libraries have a consistent "look and feel".
 
-## Trace Design
-* Trace API is described [here](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/README.md)
+This repository uses terminology (MUST, SHOULD, etc) from [RFC 2119][RFC2119].
 
-* Data model is defined [here](https://github.com/census-instrumentation/opencensus-proto/blob/master/trace/trace.proto)
+## Overview
+Today, distributed tracing systems and stats collection tend to use unique protocols and 
+specifications for propagating context and sending data to backend processing systems. This 
+is true amongst all the large vendors, and we aim to standardize these APIs and data models.
 
-## Tags Design
+The OpenCensus library refers to a collection of tools/APIs that are intended to be used primarily
+by cloud and distributed applications for monitoring and performance debugging. OpenCensus is 
```

Comment:
defocus from being first. it might not be true

OpenCensus provides a tested set of application performance management (APM) libraries, such as Metrics and Tracing, under a friendly OSS license. We acknowlege the polyglot nature of modern applications, and provide implementations in all main programming languages....

Reply:
Done.